plugin/supervisor.go

```diff
@@ -123,7 +123,7 @@ func (sup *supervisor) PerformHealthCheck() error {
 		}
 		if pingErr != nil {
 			mlog.Debug("Error pinging plugin", mlog.Err(pingErr))
-			return fmt.Errorf("Plugin RPC connection is not responding")
+			return fmt.Errorf("plugin RPC connection is not responding")
```

Comment:
Let's also return the internal error.

@larkox  - Are there any concerns if we remove the `mlog.Debug` statement? We are logging the same thing twice.

Reply:
My guts tell me that it can be removed, but I don't have the knowledge on this part of the Toolkit enough to make the decision. Maybe @lieut-data knows better or can lead to the person that can answer this?

Reply:
I can say that I've never relied on the debug log in the entirety of its existence. If we surface `pingErr` via the return value instead and it gets logged, I don't anticipate any future problems.

Reply:
Brilliant, thank you all. @cupakob - let's also remove the `mlog.Debug` statement above this.

Reply:
fixed